repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/logging/apache/format/converters/DateConversionFormat.java

```diff
@@ -19,27 +19,21 @@
  */
 package org.openrepose.commons.utils.logging.apache.format.converters;
 
-public enum DateConversionFormat {
+public class DateConversionFormat {
```

Comment:
I think this class could be re-worked in the future, but this is the pattern used elsewhere. Again, this just seemed heavyweight in the original.